lib/OpenQA/WebAPI/Controller/API/V1/Iso.pm

```diff
@@ -439,8 +463,8 @@ sub schedule_iso {
             my $job = $self->db->resultset('Jobs')->create_from_settings($settings);
             push @jobs, $job;
 
-            $testsuite_ids{_settings_key($settings)} //= [];
-            push @{$testsuite_ids{_settings_key($settings)}}, $job->id;
+            $testsuite_ids{$settings->{TEST}}->{$settings->{MACHINE}} //= [];
+            push @{$testsuite_ids{$settings->{TEST}}->{$settings->{MACHINE}}}, $job->id;
```

Comment:
There is now a outdated comment in line https://github.com/os-autoinst/openQA/pull/1813/commits/d721299482e8f2f1e0ba2910cf182cde1e990b7d#diff-9bc9bcb6a7d2b72e0fbdd65ee4792269R456. Don't know how to point to a line which wasn't changed by you :)

Reply:
changed :)